lib/plugins/attributeAlias.js

```diff
@@ -38,7 +38,10 @@ function extractSingleMapping(previous, current) {
 
     previous.direct[current.object_id] = current.name;
```

Comment:
An entry in the CNR file about this fix should be added

Reply:
Fixed in 06f744cfe57ae38e40bd57a84028f20a40bf0e53